.github/workflows/test-rc-release.yaml

```diff
@@ -25,8 +27,73 @@ on:  # yamllint disable-line rule:truthy
         default: 'main'
 
 jobs:
+  check-rc-testing-announcement:
+    runs-on: 'ubuntu-20.04'
+    if: github.event_name == 'schedule'
+    steps:
+      - name: Checkout apache-airflow
+        uses: actions/checkout@v3
+        with:
+          repository: 'apache/airflow'
+
+      - name: Parse the latest 100 GitHub issues from apache-airflow to check providers testing annocement
+        id: parse-airflow-gh-issues
+        run: |
+          # The default limit is 30. Set it to 100 for retrieving more issues
+          rc_issue_url=`gh issue list \
+            --json createdAt,title,url \
+            --limit 100 \
+            --jq 'map(
+                    select(
+                      .title |
+                      contains ("Status of testing Providers that were prepared on ")
+                    )
+                  ) | .[0].url'`
+
+          echo "rc_issue_url=$rc_issue_url" >> $GITHUB_OUTPUT
+
+      - name: Checkout current repo
+        uses: actions/checkout@v3
+        if: steps.parse-airflow-gh-issues.outputs.rc_issue_url != ''
+
+      - name: Parse the latest GitHub issues for checking existing RC provider testing issue
```

Comment:
Shouldn't we check for PRs instead of issues here? Also, where are we checking out the `astronomer-providers` repo for checking the existing PRs in our repo?

Reply:
In the current design, I'd say checking issues might make more sense as we use the timestamp as part of the branch name. But I'm thinking of renaming it as `rc-test-for-issue-$issue_number`

Reply:
If we all agree on this change, I think I'll be able to work on it this week. Or we can merge this one and create a issue for this improvement

Reply:
maybe I missing something or not understanding. Are we planning to make a change with the automation that along with PRs we will also create issues in astronomer-providers for the RCs? If yes, how are we planning to create such issues?

Reply:
Ah, it's me who misunderstood the workflow 🤦‍♂️ We actually create a PR with that title instead of the issue. Thanks for finding it out! I'll fix it late today

Reply:
okay, sorry for the late review on this one

Reply:
Just fixed it. Could you pelase check again when you have time? Thanks!